AElf.Sdk.CSharp.Tests/EventTests.cs

```diff
@@ -0,0 +1,114 @@
+using System.Security.Cryptography;
+using AElf.Common;
+using AElf.Kernel;
+using AElf.Types.CSharp;
+using Google.Protobuf;
+using Shouldly;
+using Xunit;
+
+namespace AElf.Sdk.CSharp.Tests
+{
+    public class EventTests
+    {
+        // Lot of types data should not be defined including: byte, sbyte, char, short, ushort, decimal, float and double.
+        [Fact]
+        public void Parse_EventTest()
+        {
+            var eventDemo = new EventDemo()
+            {
+               //init index value
```

Comment:
make them in different events and then check them

Reply:
have updated code and test with different events.